RELEASE-NOTES.md

```diff
@@ -4,20 +4,64 @@ section: root
 hideTOC: true
 ---
 
+# 2019.09 release notes (2019-10-29)
+Packages released:
+- @patternfly/patternfly@v2.40.1
```

Comment:
I believe this should be 2.40.2 to be in sync with the build that this will be released in.

Reply:
Updated, and also updated commit message so it will trigger a release.